contracts/StakedCelo.sol

```diff
@@ -171,4 +180,20 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
         }
         IManager(manager).transfer(from, to, amount);
     }
+
+    function _approve(
+        address owner,
+        address spender,
+        uint256 amount
+    ) internal override onlyWhenNotPaused {
+        super._approve(owner, spender, amount);
```

Comment:
`owner` is a shadowed variable. consider using `_owner`

Reply:
Changed to `from`/`to`, similar to the naming in the ERC20 spec.